### PR TITLE
Updates Readme on instructions to run pesacheck_meedan_bridge

### PR DESCRIPTION
**Pull request by koechkevin**
## Description

This PR updates Pesacheck Meedan bridge with instructions to run and execute the  script

Fixes mentioned [here](https://github.com/CodeForAfrica/api/pull/826#issuecomment-2499815258)
## Type of change

Please delete options that are not relevant.

- [x] Chore (non-breaking change which does not add visible functionality but improves code quality)
- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: pesacheck_meedan_bridge/README.md [hunk 1]
```diff
@@ -1,3 +1,41 @@
...
+To run the built docker image, execute:
+
+```sh
+docker compose --env-file ./pesacheck_meedan_bridge/.env up pesacheck_meedan_bridge
+```
+
+To run `pex` binary, execute:
+
+```sh
+docker exec -it ${container_name} ./pex
```
Comment (kilemensi) [inline pesacheck_meedan_bridge/README.md:40]: What is `${container_name}` name?

**Review by kilemensi — APPROVED** [review]

File: pesacheck_meedan_bridge/README.md (hunk 1 above)
Comment (koechkevin) [reply pesacheck_meedan_bridge/README.md]: @kilemensi can be container name or ID of the running container. I have added a script to readme  for this.

Comment (kilemensi) [reply pesacheck_meedan_bridge/README.md]: Do we need it though? why not just `docker compose exec api_pesacheck_meedan_bridge ./pex`?

**Review by DavidTheProgrammer — APPROVED** [review]
LGTM,

**Comment by kilemensi** [comment]
Lets get this over the line @koechkevin

File: pesacheck_meedan_bridge/README.md (hunk 1 above)
Comment (kilemensi) [reply pesacheck_meedan_bridge/README.md]: Why are you merging a PR with open comment/question @koechkevin ? We're using `docker compose`, we **should** be able to run it without resorting to `docker`

**Pull request merged**